mri/operators/fourier/non_cartesian.py

```diff
@@ -482,8 +482,12 @@ def __init__(self, kspace_loc, shape, implementation='cpu', n_coils=1):
         """
         self.num_slices = shape[2]
         self.shape = shape
-        (kspace_plane_loc, self.z_sample_loc,
-         self.sort_pos) = get_stacks_fourier(kspace_loc)
+        (kspace_plane_loc,
+         self.z_sample_loc,
+         self.sort_pos,
+         self.idx_mask_z) = get_stacks_fourier(
```

Comment:
I think this can be on a single line (it's just 91 characters)

Reply:
Would this be better?
```python
(kspace_plane_loc, self.z_sample_loc, self.sort_pos, self.idx_mask_z) \
            = \
            get_stacks_fourier(
                kspace_loc,
                self.shape,
            )
```

I think @zaccharieramzi  is best to suggest the right way here, but to me the above is more readable.